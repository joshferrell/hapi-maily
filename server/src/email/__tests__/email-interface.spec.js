import createEmailInterface from '../email-interface';

describe('email interface', () => {
    const nodemailer = {
        createTransport: jest.fn(() => 'hi I\'m paul')
    };

    it('should create an interface of functions to do email stuff with', () => {
        const emailSettings = {
            apiKey: 'supersecure',
            fromAddress: 'example@gmail.com'
        };

        const emailInterface = createEmailInterface(emailSettings, nodemailer);

        expect(emailInterface).toMatchSnapshot();
    });
});
