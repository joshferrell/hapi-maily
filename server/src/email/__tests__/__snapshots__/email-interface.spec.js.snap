// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`email interface should create an interface of functions to do email stuff with 1`] = `[Function]`;
